packages/fe-fpm-writer/test/unit/section.test.ts

```diff
@@ -136,7 +152,7 @@ describe('CustomSection', () => {
             expect(fs.read(fragmentPath)).toMatchSnapshot();
         });
 
-        const testVersions = [1.9, 1.85, 1.84,  1.86, 1.90];
+        const testVersions = [1.9, 1.85, 1.84, 1.86, 1.9];
```

Comment:
that looks wrong. Why `1.9` and not `1.90`, btw, now you have `1.9` twice in the array

Reply:
was a mistake. I'll undo.

Reply:
@stefanschreck  You re-requested a review but I still see the issue. Did you forget to push something?

Reply:
weird. the commit was pushed but without this correction. need a new commit.

Reply:
@tobiasqueck the change gets condensed during the commit, as 1.9 and 1.90 are the same **numbers**. I can only choose a different number, e.g. 1.98.